protocols/imap/src/main/java/org/apache/james/imap/message/request/GetAnnotationRequest.java

```diff
@@ -0,0 +1,153 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.imap.message.request;
+
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+import org.apache.james.imap.api.ImapCommand;
+
+import com.google.common.base.Preconditions;
+import org.apache.james.mailbox.model.MailboxAnnotationKey;
+
+public class GetAnnotationRequest extends AbstractImapRequest {
+    public static class Builder {
+        private String tag;
+        private ImapCommand command;
+        private String mailboxName;
+        private Set<MailboxAnnotationKey> keys;
+        private long maxsize;
```

Comment:
What is the default value for maxsize ?
I would expect an Optional here as a user can skip the maxsize parameter of the request.
This will also make the code of GetAnnotationProcessor more readable.